app/assets/stylesheets/application.scss

```diff
@@ -1 +1,111 @@
-@import "bootstrap"
+@import "bootstrap";
+
+// --- BODY --- //
+a {
+    color: #fff;
+    margin: 25px;
+}
+
+a:hover {
+    text-decoration: none;
+    color: #000;
+}
+
+h3 {
+    text-align: center;
+}
+.nav {
+    padding: 15px 0px;
+}
+
+#home-img {
+    width: 100%;
+    height: auto;
+}
+
+// --- PROFILE VIEW --- //
+#profile-contents {
+    display: flex;
+    flex-direction: wrap;
+}
+#user-profile-body {
+    text-align: center;
+    border: 1px solid #478f92;
+    margin: 15px 30px;
+    width: 500px;
+    padding: 15px 10px;
+}
```

Comment:
FYI, since we're using SCSS, rather than have long names, we can actually nest these. So like:

```
#user-profile {
  // Styles for #user-profile

  &-body {
    // Nested styles for #user-profile-body
  }
}
```